src/custom/components/swap/EthFlow/helpers.ts

```diff
@@ -184,49 +184,48 @@ export type ModalTextContentProps = {
 // returns modal content: header and descriptions based on state
 export function _getModalTextContent(params: ModalTextContentProps) {
   const { wrappedSymbol, nativeSymbol, state, isExpertMode, isNative /*, wrapSubmitted, approveSubmitted */ } = params
-  // common text
-  const swapHeader = `Swap ${wrappedSymbol}`
-  const swapDescription = `Click "Swap" to submit an off-chain transaction and swap your ${wrappedSymbol}`
-  const signatureRequiredDescription = 'Transaction signature required, please check your connected wallet'
-  const transactionInProgress = 'Transaction in progress. See below for live status updates'
+
   // wrap
-  const wrapHeader = isNative ? `Wrap your ${nativeSymbol}` : `Unwrap your ${wrappedSymbol}`
-  const wrapInstructions = `Submit an on-chain ${isNative ? 'wrap' : 'unwrap'} transaction to convert your ${
-    isNative ? nativeSymbol : wrappedSymbol
-  } into ${isNative ? wrappedSymbol : nativeSymbol}`
+  const wrapUnwrapLabel = isNative ? 'Wrap' : 'Unwrap'
+  const wrapHeader = `${wrapUnwrapLabel} your ${isNative ? nativeSymbol : wrappedSymbol} first`
+
   // approve
   const approveHeader = `Approve ${wrappedSymbol}`
-  const approveInstructions = `Give CoW Protocol permission to swap your ${wrappedSymbol} via an on-chain ERC20 Approve transaction`
+
   // both
   const bothHeader = `Wrap and approve`
-  const bothInstructions = `2 pending on-chain transactions: ${
-    isNative ? `${nativeSymbol} wrap` : `${wrappedSymbol} unwrap`
-  } and approve. Please check your connected wallet for both signature requests`
 
   let header = ''
-  let description: string | null = null
+  let descriptions: string[] | null = null
```

Comment:
I've made the descriptions an array, just to show several paragraphs. I felt there was a lot of information, so it could be separated in different paragraphs for readability.